bitnami/vault/templates/server/statefulset.yaml

```diff
@@ -230,6 +230,8 @@ spec:
               mountPath: /bitnami/vault/config
             - name: home
               mountPath: /bitnami/vault/home
+            - name: plugin
+              mountPath: /tmp              
```

Comment:
If you accept my comment below, we could delegate this to the `extraVolumeMounts`.